app/services/bill-runs/setup/year.service.js

```diff
@@ -20,7 +21,12 @@ const SessionModel = require('../../../models/session.model.js')
  */
 async function go (sessionId) {
   const session = await SessionModel.query().findById(sessionId)
-  const formattedData = YearPresenter.go(session)
+
+  const regionId = session.region
+  const twoPartTariff = session.type.startsWith('two_part')
```

Comment:
If the bill run types currently using this are
'two_part_tariff' (annual)
and
'two_part_tariff_supplementary"

And once the two_part_tariff annual bill runs are all caught up this bill run type won't be using the years service anymore. 
And the plan is for supplementary bill runs to use this eventually.

So eventually it's just:
'supplementary'
and
'two_part_tariff_supplementary"

Can we change this logic to be like this:

```suggestion
  const twoPartTariffSupplementary = session.type === 'two_part_supplementary'
```